templates/volunteers/update_profile.html

```diff
@@ -183,4 +194,4 @@ <h3 class="text-center">Update Profile</h3>
     $('#dob').datepicker("setDate", "{{ volun.dob|date:'Y-m-d' }}");
 </script>
 
-{% endblock %}
\ No newline at end of file
+{% endblock %} 
```

Comment:
No trailing whitespace should be there. There should just be a single newline at the end of the file.